scripts/update_e2e_coverage.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python3
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Updates coverage of TensorFlow e2e tests on all backends.
+
+Example usage: python3 update_e2e_coverage.py build-docs
+"""
+
+import argparse
+import collections
+import os
+import subprocess
+
+REFERENCE_BACKEND = 'tf'
```

Comment:
Nit: it would be nice to have an empty line to separate this from the following code.

Reply:
Done.